src/ontology/edits/oeo-physical.omn

```diff
@@ -2505,39 +2602,14 @@ Class: Rail
         TransportDemand
     
     
-Class: ReactivePowerplant
-
-    SubClassOf: 
-        PowerGeneratingUnit,
-        <http://purl.obolibrary.org/obo/BFO_0000051> some Turbine
-    
-    
 Class: RegenerativePowerplant
 
-    SubClassOf: 
-        PowerGeneratingUnit,
-        <http://purl.obolibrary.org/obo/BFO_0000051> some WindTurbine
-    
-    
-Class: RenewableGenerator
-
-    SubClassOf: 
-        ElectricityGenerator
-    
-    DisjointWith: 
-        FossilGenerator
-    
-    
-Class: RenewableGeneratorTechnology
-
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A renewable electricity generator technology is an electricity generator techonolgy that uses resources that are practically unlimited or regenerate quickly to generate electricity."@en
+        rdfs:comment "A RegenerativePowerplant is a Powerplant having an aggregate of one kind of RegenerativePowerUnits as its PowerGeneratingUnits."
```

Comment:
Does it really have to be one kind? I think of IEE's research on like "Kombikraftwerk".
@stap-m ?

Reply:
I agree @l-emele, it does not necessarily be of one kind. *...aggregate of RegenerativePowerUnits...* should be sufficient.

Reply:
ok I updated the affected definitions